packages/atomic/src/components/facets/atomic-date-range/atomic-date-range.tsx

```diff
@@ -15,9 +15,27 @@ export class AtomicDateRange {
    * It can be expressed as a Javascript date, as a number using epoch time or as a string using the ISO 8601 format.
    */
   @Prop() public start!: Date | string | number;
+
   /**
    * The ending date for the range.
    * It can be expressed as a Javascript date, as a number using epoch time or as a string using the ISO 8601 format.
    */
   @Prop() public end!: Date | string | number;
+
+  connectedCallback() {
+    this.validateStart();
+    this.validateEnd();
+  }
+
+  validateStart() {
+    if (!this.start) {
+      throw new Error('The date range has no start date value');
```

Comment:
I think this error message instead of "The date range with end value 2017/01/01@05:00:00 has no start value"  covers a more general case of the problem. I wanted to avoid a weird error message in the case where the user also forgot to set an end date for the end value.

Reply:
Suggestion for the message to make it easier for a dev to fix:

```
The <atomic-date-range> has no start. Please specify the "start" attribute.
```

Could also extract the throw into a dedicated function to reuse the message.

```
throwMissingDateRangeError(option: 'start' | 'end') {
    throw new Error('...');
}
```